### DISCUSSION TITLE
Request for Prebuilt Docker Images for Doctr

### DISCUSSION THREAD

**Discussion by ffalkenberg** [mindee/doctr#1297] [General]
#### Issue Description:

Hello, I've been exploring the Doctr repository and noticed that a Dockerfile is provided, which is great for containerization. However, I couldn't find any prebuilt Docker images for Doctr.

#### Questions:

1. Are there any prebuilt Docker images available for Doctr that I might have missed?
2. If not, would it be possible to provide prebuilt Docker images to simplify deployment?

Having a prebuilt Docker image would make it much easier to get started and would streamline the deployment process.

Thank you for your time and consideration.

**Comment by felixT2K** [comment]
Hi @ffalkenberg 👋🏼, 

Thanks for the request :)

Ping @odulcy-mindee maybe a request you could include into #1113  ?

**Reply by odulcy-mindee** [reply]
Yeah indeed

**Comment by odulcy-mindee** [comment]
Hello @ffalkenberg,

1. Unfortunately no, we don't have published an official Docker image for docTR
2. I'll ask internally if it's possible ;-)

**Reply by odulcy-mindee** [reply]
@ffalkenberg It should be possible, I'll take care of that!
